### PR TITLE
Add Explanation Alignment paper

### PR DESCRIPTION
**Pull request by hhybang**

### CODE REVIEW

File: _pubs/explanation-alignment.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: 'Explanation Alignment: Quantifying the Correctness of Model Reasoning At Scale'
+authors:
+  - key: hbang
+  - key: aboggust
+  - key: arvindsatya
+venue: eXCV
+year: 2024
+date: 2024-09-29
+tags:
```
Comment (arvind) [inline _pubs/explanation-alignment.md:10]: Thanks @hhybang! Could we also add a tag for `quantitative methods`?

Comment (arvind) [reply _pubs/explanation-alignment.md:10]: Oh and could we add a top-level YAML key-pair of `type: workshop` to indicate that this is a workshop paper.

File: _pubs/explanation-alignment.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title: 'Explanation Alignment: Quantifying the Correctness of Model Reasoning At Scale'
+authors:
+  - key: hbang
+  - key: aboggust
+  - key: arvindsatya
+venue: eXCV
```
Comment (arvind) [inline _pubs/explanation-alignment.md:7]: Could we add, alongside `venue`, a top-level YAML key-pair of `type: workshop` to indicate that this is a workshop paper.

File: _pubs/explanation-alignment.md (hunk 1 above)
Comment (hhybang) [reply _pubs/explanation-alignment.md:11]: Yes!

**Comment by hhybang** [comment]
Just made the updates!

**Comment by arvind** [comment]
@hhybang, I just realized, I don't think this PR includes the PDF of the paper? Could you send that along too—it should be named the same as the `.md` file, but the `.pdf` should live in the `pubs/` folder.

**Pull request merged**